### PR TITLE
Pharmacovigilance updates

### PR DESCRIPTION
**Pull request by Itskiprotich**
UHC mode dropped & unit tests updated

### CODE REVIEW

**Comment by esthermmoturi** [comment]
image
Primary contact is saved under 'People' while added members are saved under 'Household members'.Could you place them under the same card?

**Comment by esthermmoturi** [comment]
- [ ] In the `Assessment Form`, the question `**Did you notice anything strange with your medicine?**` is still asked after the user answers `No` to the question `**Are you currently taking any medication?**`
- [ ] Consider using vaccination for over 5s and immunization for u5 in this question `**Have you recently received any vaccination/immunization?**`
- [ ] Indicate relevance for all questions eg when answered No or Yes, which subsequent questions should/shouldn't appear.
- [ ] Add monthly targets (unless all time targets fit your needs)
- [ ] While logged in as a CHA/Supervisor, should I be able to create a supervisor region? Or should I just be able to create an Area?

**Review by esthermmoturi — CHANGES_REQUESTED** [review]
@Itskiprotich Kindly review the comments

**Comment by Itskiprotich** [comment]
@esthermmoturi here is my response to the above:

- [x] In the Assessment Form, the question **Did you notice anything strange with your medicine?** is still asked after the user answers No to the question **Are you currently taking any medication?**  -> The first question is under the adverse reactions while the other falls under the poor quality medicine. I've updated the form to show the sections clearly.
- [x]  Consider using vaccination for over 5s and immunization for u5 in this question **Have you recently received any vaccination/immunization?**  -> Let me have an internal discussion with the team
- [x]  Indicate relevance for all questions eg when answered No or Yes, which subsequent questions should/shouldn't appear. -> This is in place, check the assessment form the *Severe Adverse Reactions* section, 
- [x]  Add monthly targets (unless all time targets fit your needs). This has been implemented, thank you for pointing out.
- [x]  While logged in as a CHA/Supervisor, should I be able to create a supervisor region? Or should I just be able to create an Area? -> No, the supervisor region is only created at the facility level.

File: IntelliSOFT/Example CHT application/cht_pvers/translations/messages-en.properties [hunk 1]
```diff
@@ -185,24 +185,27 @@ contact.profile.type=Report Type
...
+report.assessment.reporter.group_report.group_report_adr.medication_time=Time of Medication
+report.assessment.reporter.group_report.group_report_adr.immunization_time=Time of Immunization
 report.assessment.visited_contact_uuid=Report
 
 # Analytics
 total.number.of.recoveries.reported.title=Total number of recoveries reported
 total.number.of.deaths.reported.title=Total number of deaths reported
-poor.quality.medicine.reported.title=Poor Quality Medicine Reported Identified
-adverse.drug.reactions.identified.title=Adverse Drug Reactions Identified
+poor.quality.medicine.reported.title=Poor Quality Medicine Reported Reported
```
Comment (esthermmoturi) [inline IntelliSOFT/Example CHT application/cht_pvers/translations/messages-en.properties:207]:
```suggestion
poor.quality.medicine.reported.title=Poor Quality Medicine Reported
```

**Pull request closed without merging**